airflow/www/templates/airflow/conn_edit_form.html

```diff
@@ -0,0 +1,81 @@
+{#
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+#}
+
+{% import 'appbuilder/general/lib.html' as lib %}
```

Comment:
Where did this template come from, cos it doesn't look anything like https://github.com/dpgaspar/Flask-AppBuilder/blob/master/flask_appbuilder/templates/appbuilder/general/model/edit.html

The reason I'm asking is I'm wondering if we can do it with template inheritance (where we just re-define one block and then call `{{ super() }}` to make this smaller.

Reply:
Need to override this [line](https://github.com/dpgaspar/Flask-AppBuilder/blob/master/flask_appbuilder/templates/appbuilder/general/widgets/form.html#L53)

Reply:
Might be best to  create a Form python object and add the control that way then?